Formula/dafny.rb

```diff
@@ -3,28 +3,25 @@ class Dafny < Formula
   homepage "https://github.com/dafny-lang/dafny/blob/master/README.md"
   url "https://github.com/dafny-lang/dafny/archive/v2.3.0.tar.gz"
   sha256 "ea7ae310282c922772a46a9a85e2b4213043283038b74d012047b5294687d168"
-  revision 2
+  revision 3
 
   livecheck do
     url :stable
     regex(/^v?(\d+(?:\.\d+)+)$/i)
   end
 
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "00cfdeb5892e2834b144a6e4c816a50d594440882327e65a771f9e72cd13f82d" => :catalina
-    sha256 "4b0bb8f5e2385b99318cc85ff38496e87814d5658f5dd4054fdc7d2a0a8ebc07" => :mojave
-    sha256 "4b64f7c46ab2fdfb997ca918c81e0d423609f565bdd405eaa9a2d7e848295ab7" => :high_sierra
-  end
-
   depends_on "mono-libgdiplus" => :build
   depends_on "nuget" => :build
   depends_on "mono"
-  depends_on "z3"
 
   resource "boogie" do
     url "https://github.com/boogie-org/boogie.git",
-      revision: "9e74c3271f430adb958908400c6f6fce5b59000a"
+    revision: "9e74c3271f430adb958908400c6f6fce5b59000a"
```

Comment:
Please don't remove the indentation. Indenting makes it clear that the `revision:` line is a continuation of the previous line.

Reply:
OK